frontend/src/pages/singleProduct/singleProduct.js

```diff
@@ -28,9 +38,19 @@ function SingleProduct() {
               endPoint: "Single product",
             }}
           ></PathBar>
+          <Notification
+            showNotification={showNotification}
```

Comment:
Do you need `showNotification` at all? If `showNotification` is false you will try to render component.